apps/squeaknode/docker-compose.yml

```
@@ -0,0 +1,42 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
```

The container is currently run as root, it should be run as a non-root user.

```suggestion
  web:
    user: "1000:1000"
```